src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
```

Comment:
[Don’t create several instances of the same class if you can use only one instance for your purposes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes).
**Also please use for loop for creating several objects of the same class**
For example, you need to create several users and write them in an array. In our case let’s say number of users is 3.

Bad example:
```
public class Main {
  private static final int NUMBER_OF_USERS = 3;
    
  public static void main(String[] args) {
      User[] users = new User[NUMBER_OF_USERS];
      User firstUser = new User();
      User secondUser = new User();
      User thirdUser = new User();
      users[0] = firstUser;
      users[1] = secondUser;
      users[2] = thirdUser;
  }
}
```
Improved example:
```
public class Main {
  private static final int NUMBER_OF_USERS = 3;
    
  public static void main(String[] args) {
      User[] users = new User[NUMBER_OF_USERS];
      for(int i = 0; i < users.length; i++) {
          users[i] = new User();
      }
  }
}
```